particles/Renderer.cpp

```diff
@@ -75,31 +75,19 @@ void Renderer::init(score::gfx::RenderList& renderer)
     auto& n = static_cast<const Node&>(this->node);
     auto& rhi = *renderer.state.rhi;
 
-    std::cout << "started init renderer\n";
-    const TexturedMeshForParticles mesh(n.meshName);
```

Comment:
This was incorrect: this would have created the mesh *only* for the duration of this function, it would have been destroyed afterwards => crash.
The instances have to be class members at least.